### PR TITLE
Feature/scd40

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+namespace Meadow.Foundation.Sensors.Environmental
+{
+    /// <summary>
+    /// Base class for SCD4x series of C02 sensors
+    /// </summary>
+    public abstract partial class Scd4x : ByteCommsSensorBase<(Concentration? Concentration, 
+                                                        Units.Temperature? Temperature,
+                                                        RelativeHumidity? Humidity)>,
+        ITemperatureSensor, IHumiditySensor
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs:16]: Dont we have a IConcentrationSensor?

File: Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+        /// <summary>
+        /// Raised when the humidity value changes
+        /// </summary>
+        public event EventHandler<IChangeResult<RelativeHumidity>> HumidityUpdated = delegate { };
+
+        /// <summary>
+        /// The current C02 concentration value
+        /// </summary>
+        public Concentration? Concentration { get; private set; }
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs:36]: public Concentration? Concentration => Conditions.Concentration; ?

**Review by jorgedevs — CHANGES_REQUESTED** [review]

File: Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs:16]: We don't and I was on the fence about adding it ... I'll do it quickly now

File: Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Environmental.Scd4x/Driver/Scd4x.cs:36]: yes! thx - I'll fix

**Pull request merged**